tendrl_gluster_peers_bricks.yml

```diff
@@ -1,9 +1,10 @@
 ---
 #
-# * Install Tendrl server and tendrl-gluster nodes.
-# * Configure gluster trusted storage pool and create bricks 
-#   on all available devices on gluster nodes.
+# Install Tendrl server and tendrl-gluster nodes.
+# Configure gluster trusted storage pool and create bricks
+# on all available devices on gluster nodes.
 #
 - include: qe_pre_installation_tasks.yml
-- include: tendrl_gluster.yml
```

Comment:
Just a question. Why is `tendrl.yml` and `gluster.yml` used instead of `tendrl_gluster.yml` that also includes them? Is there some reason for keeping `tendrl_gluster.yml` playbook? Maybe if you would drop this playbook then the setup infrastructure would be simpler...